test/info.py

```diff
@@ -125,6 +126,7 @@
     (   0x7404,     'lpc11u35_6lowpan_borderrouterethernet_if', None,               'mbed-6LoWPAN-Border-Router-ETHERNET'   ),
     (   0xC000,     'lpc11u35_cocorico_if',                     None,               None                                    ), # TODO - set target to 'CoCo-ri-Co' when mbed-os supports this
     (   0x5500,     'lpc11u35_gr_peach_if',                     None,               'Renesas-GR-PEACH'                      ),
+    (   0x6660,     'lpc11u35_nz32_sc151_if',                   None,               'NZ32-SC151'                            ),
```

Comment:
The target name in mbed-os is "NZ32_SC151". It is listed in "targets.json" file here:
https://github.com/ARMmbed/mbed-os/blob/master/targets/targets.json#L1337
The platform page has also already been added to mbed, but not public yet:
https://developer.mbed.org/platforms/nz32-sc151/
For example, to compile project via mbed-cli using mbed-os, the following works:
mbed compile -t GCC_ARM -m NZ32_SC151